css/styles.css

```diff
@@ -215,7 +215,6 @@ body {
 }
```

Comment:
**Kindly update your link**
- Your link send to a page not found in your README file.